test/fixtures/challenges/challenges.js

```diff
@@ -0,0 +1,10 @@
+module.exports = () => {
+  return [
+    {
+      title: "Sherlock and Anagrams",
+      level: "Easy",
+      start_date: parseInt(Date.now() / 1000),
+      end_date: parseInt(Date.now() / 1000) + 60 * 60 * 24 * 10, // After 10 days
```

Comment:
**Suggestion**: Add a constant variable to store seconds
Reference - https://github.com/ryanmcdermott/clean-code-javascript#use-searchable-names